### ISSUE TITLE
Install Failure with 1.0.3

### ISSUE THREAD

**Issue by quasiben** [dask/dask-labextension#80]
This could very well be a user error but i am having trouble naively installing dask-labextension:

```
jlab-1.0) bzaitlen@dgx15:~/GitRepos/jupyterlab-nvdashboard$ pip install dask_labextension
(jlab-1.0) bzaitlen@dgx15:~/GitRepos/jupyterlab-nvdashboard$ jupyter labextension install dask-labextension
An error occured.
KeyError: '@jupyterlab/apputils'
```

Useful dependencies

```
(jlab-1.0) bzaitlen@dgx15:~/GitRepos/jupyterlab-nvdashboard$ conda list jupyter
# packages in environment at /home/nfs/bzaitlen/miniconda3/envs/jlab-1.0:
#
# Name                    Version                   Build  Channel
jupyter-server-proxy      1.1.0                      py_0    conda-forge
jupyter_client            5.3.1                      py_0    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
jupyterlab                1.0.8                    py37_0    conda-forge
jupyterlab_server         1.0.6                      py_0    conda-forge
```

**Comment by ian-r-rose** [comment]
There was a publish of JupyterLab this morning (1.0.8) that was broken and had errors like this. It has since been fixed (1.0.9). Can you try updating?

**Comment by quasiben** [comment]
Yes, that was it.  Thank you
